tests/system/python/api/test_audit.py

```diff
@@ -110,5 +130,11 @@ def test_create_audit_entry(self, fledge_url, payload, total_count):
         assert 200 == r.status
         r = r.read().decode()
         jdoc = json.loads(r)
+
+        # sqlite has an additional entry: CONAD, INFORMATION, '{{"name":"sqlite", ...}'
+        # so total_count is being decresed in case of 'postgres' storage engine
+        if storage_plugin == 'postgres':
+           total_count = total_count -1
```

Comment:
this is okay!

Reply:
we can put `test_create_audit_entry` fix in specific pull request and can merge this only fix (and a separate pull request for api.test_audit.TestAudit.test_default_get_audit)